src/design/optForce/analizeOptForceSol.m

```diff
@@ -0,0 +1,131 @@
+function [maxGrowthRate, minTarget, maxTarget] = analizeOptForceSol(model, targetRxn, solution, relax, tol)
```

Comment:
I am using MATLAB.devTools as suggested by Ronan. Exist that in MATLAB.devTools?